### PR TITLE
sync zh-trans /docs/concepts/workloads/pods/pod.md

### PR DESCRIPTION
**Pull request by tanjunchen**
sync zh-trans /docs/concepts/workloads/pods/pod.md

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 1]
```diff
@@ -30,151 +27,152 @@ _Pods_ 是可以在 Kubernetes 中创建和管理的、最小的可部署的计
...
 containers which are relatively tightly coupled &mdash; in a pre-container
 world, being executed on the same physical or virtual machine would mean being
 executed on the same logical host.
 -->
-
-## Pod 是什么？
-
-_pod_ （就像在鲸鱼荚或者豌豆荚中）是一组（一个或多个）容器（例如 Docker 容器），这些容器有共享存储、网络、以及怎样运行这些容器的声明。Pod 的内容总是共同定位和共同调度，并且在共享的上下文中运行。
-Pod 以特定于应用的“逻辑主机”为模型，它包含一个或多个应用容器，这些容器是相对紧密的耦合在一起；在容器出现之前，在相同的物理机或虚拟机上运行意味着在相同的逻辑主机上运行。
+_Pod_ （就像在鲸鱼荚或者豌豆荚中）是一组（一个或多个）{{< glossary_tooltip text="容器" term_id="container" >}}（例如 Docker 容器），这些容器有共享存储、网络、以及怎样运行这些容器的声明。Pod 的内容总是共同定位和共同调度，并且在共享的上下文中运行。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
_Pod_ （就像在鲸鱼荚或者豌豆荚中）是一组（一个或多个）{{< glossary_tooltip text="容器" term_id="container" >}}（例如 Docker 容器），这些容器共享存储、网络、以及怎样运行这些容器的声明。Pod 中的内容总是并置（colocated）的并且一同调度，在共享的上下文中运行。
```

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 2]
```diff
@@ -30,151 +27,152 @@ _Pods_ 是可以在 Kubernetes 中创建和管理的、最小的可部署的计
...
+Pod 以特定于应用的“逻辑主机”为模型，它包含一个或多个应用容器，这些容器是相对紧密的耦合在一起 &mdash; 在容器出现之前，在相同的物理机或虚拟机上运行意味着在相同的逻辑主机上运行。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
Pod 所建模的是特定于应用的“逻辑主机”，其中包含一个或多个应用容器，这些容器是相对紧密的耦合在一起 &mdash; 在容器出现之前，在相同的物理机或虚拟机上运行意味着在相同的逻辑主机上运行。
```

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 3]
```diff
@@ -30,151 +27,152 @@ _Pods_ 是可以在 Kubernetes 中创建和管理的、最小的可部署的计
...
 
 <!--
 While Kubernetes supports more container runtimes than just Docker, Docker is
-the most commonly known runtime, and it helps to describe pods in Docker terms.
+the most commonly known runtime, and it helps to describe Pods in Docker terms.
+-->
+虽然 Kubernetes 支持多种容器运行时，但 Docker 是最常见的一种运行时，它有助于描述 Docker 术语中的 Pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
虽然 Kubernetes 支持多种容器运行时，但 Docker 是最常见的一种运行时，它有助于使用 Docker 术语来描述 Pod。
```

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 4]
```diff
@@ -30,151 +27,152 @@ _Pods_ 是可以在 Kubernetes 中创建和管理的、最小的可部署的计
...
+handled automatically for containers in a Pod.
 -->
-
-## Pod 的动机
-
-### 管理
-
-Pod 是形成内聚服务单元的多个协作过程模式的模型。它们提供了一个比它们的应用组成集合更高级的抽象，从而简化了应用的部署和管理。Pod 可以用作部署单元、水平缩放和复制。在 Pod中，多个容器的共同定位（协同调度）、共享命运（例如，终止）、协调复制、资源共享和依赖项管理都是自动处理的。
-
+Pod 是形成内聚服务单元的多个协作过程模式的模型。它们提供了一个比它们的应用组成集合更高级的抽象，从而简化了应用的部署和管理。Pod 可以用作部署、水平扩展和副本的最小单元。在 Pod 中，多个容器的共同定位（协同调度）、共享命运（例如，终止）、协调副本、资源共享和依赖项管理都是自动处理的。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
Pod 是形成内聚服务单元的多个协作过程模式的模型。它们提供了一个比它们的应用组成集合更高级的抽象，从而简化了应用的部署和管理。Pod 可以用作部署、水平扩展和制作副本的最小单元。在 Pod 中，系统自动处理多个容器的在并置运行（协同调度）、生命期共享（例如，终止），协同复制、资源共享和依赖项管理。
```

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 5]
```diff
@@ -185,10 +183,7 @@ programs, such as:
 * proxies, bridges, and adapters
 * controllers, managers, configurators, and updaters
 -->
-
-## 使用 Pod
-
-Pod 可以用于托管垂直集成的应用程序栈（例如，LAMP），但最主要的动机是支持位于同一位置的、共同管理的帮助程序，例如：
+Pod 可以用于托管垂直集成的应用程序栈（例如，LAMP），但最主要的目的是支持位于同一位置的、共同管理的帮助程序，例如：
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]: helper programs -> 工具程序？

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 6]
```diff
@@ -198,24 +193,27 @@ Pod 可以用于托管垂直集成的应用程序栈（例如，LAMP），但最
 
 
 <!--
-Individual pods are not intended to run multiple instances of the same
+Individual Pods are not intended to run multiple instances of the same
 application, in general.
+-->
+通常，单个 Pod 并不打算运行同一应用程序的多个实例。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
通常，不会用单个 Pod 来运行同一应用程序的多个实例。
```

File: content/zh/docs/concepts/workloads/pods/pod.md [hunk 7]
```diff
@@ -239,36 +234,37 @@ _为什么不在单个（Docker）容器中运行多个程序？_
 
 <!--
 _Why not support affinity-based co-scheduling of containers?_
+-->
+_为什么不支持基于亲和性的容器联合调度？_
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod.md]:
```suggestion
_为什么不支持基于亲和性的容器协同调度？_
```

File: content/zh/docs/concepts/workloads/pods/pod.md (hunk 5 above)
Comment (tanjunchen) [reply content/zh/docs/concepts/workloads/pods/pod.md]: better

**Comment by tanjunchen** [comment]
done

**Pull request merged**